cukas.js

```diff
@@ -170,6 +196,21 @@ class CukasGame {
           return 0;
       }
     }
+
+    toJSON(){
+        return {
+            "type": "cukasGame",
+            "gameState": this.gameState,
+            "players": this.players,
+            "deck": this.deck,
+            "playerCount": this.playerCount,
+            "trumpCard": this.trump,
+            "attackCards": this.attack,
+            "defenseCards": this.defence,
```

Comment:
`"defenseCards" should be "defen**c**eCards"

Reply:
Goddamn British English.

Reply:
More like goddamn americans smh